tests/main/store-state/task.yaml

```diff
@@ -40,6 +29,10 @@ execute: |
         snap info core | MATCH "store-url:.*https://snapcraft.io"
     fi
 
+    # install dependecies before switching to fake store
```

Comment:
This seems related to the base change, but I don't really understand the significance. Can we add this in a second pass later?

Reply:
Not really, fake store does not forward install requests to the real store. The test snap installed here had no base, hence defaulted to 'core'. We installed jq & remarshal both of which pulled in 'core'. Since the install happened before switching over to fake store, it all worked by accident as the dependencies happened to be satisfied. Without jq or remarshal nothing pulls in the dependencies. That's why the snap's base was switched to core22 too, just to avoid pulling in core.